dashboard/agent.py

```diff
@@ -152,7 +157,42 @@ async def _check_parent():
                 while True:
                     parent = curr_proc.parent()
                     if parent is None or parent.pid == 1 or self.ppid != parent.pid:
-                        logger.error("Raylet is dead, exiting.")
+                        log_path = "/tmp/ray/session_latest/logs/raylet.out"
+                        error = False
+                        msg = f"Raylet is terminated: ip={self.ip}, id={self.node_id}. "
+                        try:
+                            with open(log_path, "r") as f:
+                                raylet_logs = f.readlines()
```

Comment:
Are we reading the whole file? Maybe we can read last X lines here as well?

Reply:
Sure, reading last 1MB if the log file is larger than that.

Reply:
Can you add a test for this new update?

Reply:
sure